### PR TITLE
docs(project): document workflow

### PR DESCRIPTION
**Pull request by dimaip**

### CODE REVIEW

**Comment by dimaip** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,51 +1,42 @@
...
-New feature: `feature/{feature_name}#{number}`
+3. Follow Github flow: https://guides.github.com/introduction/flow/
 
-#### Examples:
 
-  * `feature/hint`
-  * `feature/massage_box#123`
+* Follow this convention for branch naming: `<type>/<short_name>#<issue_number>`.
 
+Where `<type>` may be among: `feat`, `fix`, `docs`, `refactor`, `test`, `chore` (build changes).
```
Comment (Nitive) [inline README.md]: type is always `feature`

Comment (Nitive) [inline README.md]: or it would be `docs` maybe

Comment (Nitive) [inline README.md]: Nobody would create branch for refactor, chore and test

Comment (dimaip) [inline README.md]: Why? I think it should follow issue types

**Pull request merged**